CommonTools/MVAUtils/interface/TMVAEvaluator.h

```diff
@@ -0,0 +1,50 @@
+#ifndef CommonTools_MVAUtils_TMVAEvaluator_h
+#define CommonTools_MVAUtils_TMVAEvaluator_h
+
+#include <map>
+#include <memory>
+#include <mutex>
+#include <string>
+#include <vector>
+
+#include "CondFormats/EgammaObjects/interface/GBRForest.h"
+#include "FWCore/Framework/interface/EventSetup.h"
+#include "FWCore/Utilities/interface/thread_safety_macros.h"
+#include "TMVA/IMethod.h"
+#include "TMVA/Reader.h"
+
+class TMVAEvaluator {
```

Comment:
this will collide with the same class name defined in CommonTools/Utils/interface/TMVAEvaluator.h